client/app/components/Resource/index.jsx

```diff
@@ -11,14 +11,15 @@ type Props = {
   tags?: string[],
   tagged?: boolean,
   title: string,
+  updateTagFilter?: Function,
 };
 
-const taggedResources = (tagged: ?boolean, tags: ?(string[])) => {
+const taggedResources = (tagged: ?boolean, tags: ?(string[]), updateTagFilter) => {
   if (tagged && tags) {
     return (
       <div className="tags">
         {tags.map((tag) => (
-          <Tag normal label={tag} key={Utils.randomString()} />
+          <Tag normal label={tag} key={Utils.randomString()} onClick={ (tagLabel) => { updateTagFilter(tagLabel) } } />
```

Comment:
You can re-write this as `onClick={updateTagFilter ? (tagLabel) => updateTagFilter(tagLabel) : null}`. I think this is what is responsible for the Flow error that's happening in the CircleCI build.